library/src/main/java/com/datatorrent/lib/projection/ProjectionOperator.java

```diff
@@ -308,5 +303,45 @@ private void handleProjection(Object t)
     }
   }
 
+  /**
+   * set selectFields, a list of fields to be selected from incoming POJO
+   *
+   * @param selectFields List of fields from POJO to be selected
+   */
+  public void setSelectFields(List<String> selectFields)
```

Comment:
Now that we have proper configuration for selectFields, can you please add @useSchema and @description javadoc metatag. You can refer to POJOEnricher setIncludeFields setter method for this.